### ISSUE TITLE
upgrade to pytorch 1.5

### ISSUE THREAD

**Issue by martinjaggi** [mlbench/mlbench-core#74]
(and measure performance impact compared to earlier mlbench versions)

https://github.com/pytorch/pytorch/releases/tag/v1.5.0

and could potentially check if the channels-last memory format has any effect here for the cifar10 benchmark

**Comment by giorgiosav** [comment]
I ran the 2 CIFAR10 tasks on NCCL with 2 workers and saw some speedup compared to Pytorch1.4, although I don't know if it's meaningful on such short runs... 
I also tried running the linear task but it got interrupted for some reason, so I'm trying again.

CIFAR10, Pytorch 1.4
`80% Top 1 Validation Accuracy reached in 71.211 seconds, Compute: 38.51862716674805 seconds, Communication: 30.199332237243652 seconds`

CIFAR10, Pytorch 1.5
`80% Top 1 Validation Accuracy reached in 42.651 seconds, Compute: 24.443291187286377 seconds, Communication: 16.425377368927002 seconds`

CIFAR10 scaling, Pytorch 1.4
`80% Top 1 Validation Accuracy reached in 71.905 seconds, Compute: 26.496683359146118 seconds, Communication: 43.847965240478516 seconds`

CIFAR10 scaling, Pytorch 1.5
`80% Top 1 Validation Accuracy reached in 63.596 seconds, Compute: 25.506200551986694 seconds, Communication: 36.541961431503296 seconds`

**Comment by giorgiosav** [comment]
And for the Linear benchmark, on 2 workers with NCCL:
Pytorch1.4
`89% Top 1 Validation Accuracy reached in 73.740 seconds, Compute: 59.28000283241272 seconds, Communication: 12.621702671051025 seconds`

Pytorch1.5
`89% Top 1 Validation Accuracy reached in 34.436 seconds, Compute: 26.36866021156311 seconds, Communication: 7.182677745819092 seconds`
